src/css/protocol/layouts/columns.scss

```diff
@@ -2,7 +2,5 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-$font-path: '/static/protocol/fonts';
-$image-path: '/static/protocol/img';
-
+@use '~@mozilla-protocol/core/protocol/css/includes/lib';
```

Comment:
@craigcook I'm assuming this one is fine without the explicit image support, because the columns CSS is purely structural with no assets, yeah?